test/ca/mcgill/jetuml/layouttests/AbstractTestDiagramLayout.java

```diff
@@ -108,9 +115,66 @@ protected Edge edgeByMiddleLabel(String pLabel)
 				.collect(Collectors.toUnmodifiableList());
 	}
 	
+	protected static int getStaticIntFieldValue(Class<?> pClass, String pFieldName) throws ReflectiveOperationException
+	{
+		Field field = pClass.getDeclaredField(pFieldName);
+		field.setAccessible(true);
+		int fieldValue = field.getInt(null);
+		return fieldValue;
+	}
+	
+	private static void verifyDefaultDimensions(Node pNode, int pDefaultWidth, int pDefaultHeight)
+	{
+		Rectangle bounds = NodeViewerRegistry.getBounds(pNode);
+		assertEquals(pDefaultWidth, bounds.getWidth());
+		assertEquals(pDefaultHeight, bounds.getHeight());
+	}
+	
 	protected static void verifyPosition(Node pNode, int pExpectedX, int pExpectedY)
 	{
 		assertEquals(pExpectedX, pNode.position().getX());
 		assertEquals(pExpectedY, pNode.position().getY());
 	}
+	
+	protected static void verifyClassNodeDefaultDimensions(Node pNode)

```

Comment:
This would be better pushed down into an intermediate abstract class `AbstractTestClassDiagramLayout`, since it's irrelevant to non-class-diagrams. Same idea for `verifyObjectNode...`